### DISCUSSION TITLE
can't use PushType "DeleteThenCreate"

### DISCUSSION THREAD

**Discussion by chandrasajana** [BHoM/BHoM#1613] [Q&A]
I have an issue when updating the model to ETABS from GH using _PushType_ "DeleteThenCreate", except for this option everything seems working fine.

![Screenshot 2024-04-29 000920](https://github.com/BHoM/BHoM/assets/93826774/516a724e-2815-44ab-bcf7-9780a4af2d35)

any tips and insights for this issue would be greatly appreciated. Thank you.

**Comment by samuelbasimalla** [comment]
Hi @chandrasajana 
ETABS toolkit currently only supports creating and reading bar elements. 'Update' and 'Delete' are not supported at the moment.
![image](https://github.com/BHoM/BHoM/assets/148187206/746b8ed7-8110-4a23-b597-053c68fbe9e2)

For full list of what ETABS toolkit supports currently, please refer below:
https://github.com/BHoM/ETABS_Toolkit/wiki/BHoM---ETABS-object-relations

**Reply by chandrasajana** [reply]
hopefully BHoM dev will add this feature in the next release, which will make big changes to the modelling process. Thank you for the explanation

**Reply by NicoRamDLR** [reply]
having the ability to delete elements is necessary for performing large model modifications. would be great to have. thanks!
